go/vt/vttablet/tabletserver/tabletserver_test.go

```diff
@@ -154,9 +154,10 @@ func TestTabletServerPrimaryToReplica(t *testing.T) {
 	// Reuse code from tx_executor_test.
 	_, tsv, db := newTestTxExecutor(t, ctx)
 	// This is required because the test is verifying that we rollback transactions on changing serving type,
-	// but that only happens immediately if the shut down grace period is not specified.
-	tsv.te.shutdownGracePeriod = 0
-	tsv.sm.shutdownGracePeriod = 0
+	// but that only happens when we have a shutdown grace period, otherwise we wait for transactions to be resolved
+	// indefinitely.
+	tsv.te.shutdownGracePeriod = 1
+	tsv.sm.shutdownGracePeriod = 1
 	defer tsv.StopService()
 	defer db.Close()
```

Comment:
I believe removing this change should be enough as the default configuration is 3 seconds

Reply:
test only waits for 100 ms and the default is 3 seconds. so this makes the test run faster